### PR TITLE
update pagination display

### PR DESCRIPTION
**Pull request by eregnier**
Try to solve #107

### CODE REVIEW

File: ideasbox/static/ideasbox/main.css [hunk 1]
```diff
@@ -505,6 +505,22 @@ html[dir='rtl'] .card img {
     background-repeat: no-repeat;
     background-position: center right;
 }
+
+.smallButton {
```
Comment (yohanboniface) [inline ideasbox/static/ideasbox/main.css]: what about factorizing with `.button`?

File: ideasbox/templates/ideasbox/pagination.html [hunk 2]
```diff
@@ -2,13 +2,24 @@
 {% if is_paginated %}
     <div class="pagination">
         {% if page_obj.has_previous %}
-            <a href="{{ base_url }}?page={{ page_obj.previous_page_number }}" class="previous">{% trans "previous" %}</a>
+
+            {% if page_obj.previous_page_number != 1 %}
```
Comment (yohanboniface) [inline ideasbox/templates/ideasbox/pagination.html]: this seems to break some tests. Maybe this is the test to be fixed, as the change looks OK.

Comment (eregnier) [inline ideasbox/templates/ideasbox/pagination.html]: python manage.py test 
does not reproduce the error. How can I produce the failing test on my side please ?

Comment (yohanboniface) [inline ideasbox/templates/ideasbox/pagination.html]: run `py.test` :)

Comment (eregnier) [inline ideasbox/templates/ideasbox/pagination.html]: I Have no particular problems with unit tests
http://pastebin.com/09bb7MPE

Comment (yohanboniface) [inline ideasbox/templates/ideasbox/pagination.html]: Ran 0 tests in 0.000s

**Pull request closed without merging**